packages/cli/package.json

```diff
@@ -48,6 +48,7 @@
     "cors": "^2.8.5",
     "dag-jose": "^0.3.0",
     "did-resolver": "^3.1.0",
+    "did-safe-resolver": "^0.0.4",
```

Comment:
Package should be called `safe-did-resolver`.

Reply:
Can we contact the author here to get the name? https://www.npmjs.com/package/safe-did-resolver